docs/transport-udfs-trino.patch

```diff
@@ -1,23 +1,23 @@
-diff --git a/presto-main/src/main/java/io/prestosql/server/PluginManager.java b/presto-main/src/main/java/io/prestosql/server/PluginManager.java
+diff --git a/trino-main/src/main/java/io/trino/server/PluginManager.java b/trino-main/src/main/java/io/trino/server/PluginManager.java
```

Comment:
I tried to apply this patch to Trino release tag 352 and the apply failed.

```
$ ~/trino @ 690ddcd090 > git status
HEAD detached at 352
nothing to commit, working tree clean
$ ~/trino @ 690ddcd090 > git apply ~/transport/docs/transport-udfs-trino.patch
error: corrupt patch at line 48
```
Can you take a look?

Reply:
thanks for verifying! fixed the patch